smac/stats/stats.py

```diff
@@ -28,18 +31,57 @@ def __init__(self, scenario):
         self._n_configs_per_intensify = 0
         self._n_calls_of_intensify = 0
         ## exponential moving average
-        self._ema_n_configs_per_intensifiy = 0
+        self._ema_n_configs_per_intensify = 0
         self._EMA_ALPHA = 0.2
 
         self._start_time = None
         self._logger = logging.getLogger(self.__module__ + "." + self.__class__.__name__)
 
+    def save(self):
+        """
+        Save all relevant attributes to json-dictionary.
+        """
+        data = {'ta_runs': self.ta_runs,
```

Comment:
Can't we save the Scenario attributes in a similar generic way as in `load()`?
Such a hand-written dictionary is also hard to maintain.